src/include/cpu/rpp_cpu_simd.hpp

```diff
@@ -279,6 +279,66 @@ inline RppStatus rpp_store48_f32pln3_to_u8pkd3(Rpp8u *dstPtr, __m128 *p)
     return RPP_SUCCESS;
 }
 
+inline RppStatus rpp_load48_u8pkd3_to_u8pln3(Rpp8u *srcPtr, __m128i *px)
+{
+    __m128i pxSrc[8];
+    __m128i pxMask = _mm_setr_epi8(0, 3, 6, 9, 1, 4, 7, 10, 2, 5, 8, 11, 12, 13, 14, 15);
```

Comment:
I think pxMask and pxMaskRGB is repeatedly used in many functions. Consider defining it as global constant